translations/en-us.yaml

```diff
@@ -4365,6 +4365,18 @@ clusterNew:
       versionHelp: Verrazzano Version
       versionOverride: Verrazzano Version Override
       tagOverride: Verrazzano Tag Override
+    imagePull:
+      create: Create image Pull Secret
+      title: Image Pull Secret (Optional)
+      detail: Provide inputs for creating an image pull secret to download Verrazzano and related images from a secured registry
+      registry: Registry server
```

Comment:
looks like some of these are not used and can be removed.

Reply:
Removed